sdk/containerregistry/azure-containerregistry/azure/containerregistry/_helpers.py

```diff
@@ -27,6 +27,8 @@
 
 # Public cloud audience
 AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD = "https://management.azure.com"
```

Comment:
nit: does this need to be deleted since its gone from elsewhere too ?

Reply:
Yeah, I'll remove it as it's just used in tests now.